### ISSUE TITLE
Problems when testing flow stream model

### ISSUE THREAD

**Issue by mangdian** [yjxiong/temporal-segment-networks#91]
`CUDA_VISIBLE_DEVICES=1 python tools/eval_net.py ucf101 1 flow data/Thumos/test/ models/ucf101/tsn_bn_inception_rgb_deploy.prototxt models/ucf101_split_1_tsn_rgb_reference_bn_inception.caffemodel  --num_worker 1 --save_scores data/test.npz
`
I run the command above,and it looks fine when loading caffemodel,and get this error when the model is loaded.
`
I0501 17:12:55.414881 17891 net.cpp:285] This network produces output fc-action
I0501 17:12:55.415419 17891 net.cpp:551] Collecting Learning Rate and Weight Decay.
I0501 17:12:55.415740 17891 net.cpp:300] Network initialization done.
I0501 17:12:55.415782 17891 net.cpp:301] Memory required for data: 74005652
F0501 17:12:55.507350 17891 base_conv_layer.cpp:117] Check failed: bottom[0]->channels() == channels_ (10 vs. 3) Input size incompatible with convolution kernel.
*** Check failure stack trace: ***
Aborted (core dumped)
`
how can I fix this?
Thank you for your help.

**Comment by yjxiong** [comment]
`tsn_bn_inception_rgb_deploy.prototxt `
